modules/sc-mesh-secure-deployment/src/2_0/features/jamming/preprocessor.py

```diff
@@ -0,0 +1,169 @@
+import json
+import os
+import sys
+from typing import Optional, Tuple
+
+import numpy as np
+import pandas as pd
```

Comment:
are these available in Host OS with properly installed?

Reply:
yes

Reply:
without tar/zip hack package?

Reply:
Currently these are installed with the tar/zip hack

Reply:
Needs to be installed proper way in a Host OS during build time and not in a device start-up.